manager/garden_manager.py

```diff
@@ -59,7 +64,62 @@ def find_all_with_orchard(self) -> list:
         """
         return [garden for garden in self.gardens if garden.has_orchard()]
 
-    def __str__(self):
+    def get_lsit_of_has_orchard(self) -> list[bool]:
+        """
+        Returns a list of boolean values indicating whether each garden in manager has an orchard.
+
+        Returns:
+            list[bool]: A list of boolean values.
+                        indicating whether the garden has an orchard (True) or not (False).
+        """
+        return [garden.has_orchard() for garden in self.gardens]
+
+    def get_enumerate(self):
+        """
+        Get the enumerated list of gardens.
+
+        Returns:
+            list: A list of tuples containing the index and garden object.
+        """
+        return list(enumerate(self.gardens))
```

Comment:
I think no need to cast it to list now. If you use it, you would need iterator, not necessarily a list. If you want a list, you can convert it to list right there where you need it